### PR TITLE
Odd and even numbers exercise

### PR DESCRIPTION
**Pull request by anciuvaga**

### CODE REVIEW

File: src/javaExercises/oddEvenNumbers/OddEvenNumbers.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    public static void findNumbers(int[] arr) {
+        int amountEvenNumbers = 0;
+        int sumOddNumbers = 0;
+
+        for (int i = 0; i < arr.length; i++) {
+
+            if (arr[i] % 2 == 0) {
+                amountEvenNumbers++;
+            } else {
+                sumOddNumbers += arr[i];
```
Comment (CristianManic) [inline src/javaExercises/oddEvenNumbers/OddEvenNumbers.java]: Let's separate all this calculations in 2 separate methods, one will return the sum of add numbers and the other one will return the amount of even numbers

Comment (anciuvaga) [reply src/javaExercises/oddEvenNumbers/OddEvenNumbers.java]: Done

File: src/javaExercises/oddEvenNumbers/OddEvenNumbers.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        return amountEvenNumbers;
+    }
+
+    public static int calcOddNumbers(int[] arr){
+        int sumOfOddNumbers = 0;
+        for (int i = 0; i < arr.length; i++) {
+
+            if (arr[i] % 2 != 0)
+                sumOfOddNumbers += arr[i];
+        }
```
Comment (CristianManic) [inline src/javaExercises/oddEvenNumbers/OddEvenNumbers.java]: try to implement this using lambda

File: src/javaExercises/oddEvenNumbers/OddEvenNumbers.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+public class OddEvenNumbers {
+
+    public static int calcEvenNumbers(int[] arr){
+       int amountEvenNumbers = 0;
+        for (int i = 0; i < arr.length; i++) {
+
+            if (arr[i] % 2 == 0)
+                amountEvenNumbers++;
+        }
+        return amountEvenNumbers;
```
Comment (CristianManic) [inline src/javaExercises/oddEvenNumbers/OddEvenNumbers.java]: same here

File: src/javaExercises/oddEvenNumbers/OddEvenNumbers.java (hunk 2 above)
Comment (anciuvaga) [reply src/javaExercises/oddEvenNumbers/OddEvenNumbers.java]: Done

File: src/javaExercises/oddEvenNumbers/OddEvenNumbers.java (hunk 3 above)
Comment (anciuvaga) [reply src/javaExercises/oddEvenNumbers/OddEvenNumbers.java]: Done

File: src/javaExercises/oddEvenNumbers/OddEvenNumbers.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
+package javaExercises.oddEvenNumbers;
+
+import java.util.Arrays;
+
+public class OddEvenNumbers {
+    public static void findNumbers(int[] arr) {
+        System.out.println("Elements of the array are: " + Arrays.toString(arr));
+        System.out.println("In arrey are: " + Arrays.stream(arr).filter(x -> x % 2 == 0).count()
```
Comment (CristianManic) [inline src/javaExercises/oddEvenNumbers/OddEvenNumbers.java]: please change it back to perform it in two separate methods with an return type and print element from outside, you can add a short method that will print any array

Comment (anciuvaga) [reply src/javaExercises/oddEvenNumbers/OddEvenNumbers.java]: Done

**Review by CristianManic — APPROVED** [review]

**Pull request merged**